analyzer/asn1.spicy

```diff
@@ -92,7 +92,8 @@ type ASN1Tag = unit {
   var class: ASN1Class;
   var constructed: bool;
 
-  : bitfield(8) {
+  # TODO(bbannier): Use an anonymous bitfield here once zeek/spicy#1533 is fixed.
```

Comment:
This isn't related to #1533, is it?

Reply:
It is indirectly. With spicy-1.9 I would want to use an anonymous bitfield again and remove the unit vars. This then fails when an `ASN1Tag` is used elsewhere (here: passed as a unit parameter exactly as in zeek/spicy#1533).